### PR TITLE
Add a `.phylum_project` file for GitHub App

### PR DESCRIPTION
**Pull request by kylewillmon**
This is a temporary solution to convince the GitHub App to ignore the many lockfiles in this repository that are used for testing. Those lockfiles contain many issues that would be concerning if they were actually in use.

### CODE REVIEW

**Review by maxrake — APPROVED** [review]
Would adding these lines to a `.phylum_project` file that is produced by `phylum init` also work? Approving now since this is an admitted temporary fix.

**Comment by cd-work** [comment]
I don't understand this fix, this isn't actually a valid `.phylum_project` file, is it? Is there a reason why you didn't go with the proper format?

**Comment by kylewillmon** [comment]
Putting this on hold due to ongoing discussions about the file format and difference in functionality between the CLI and GitHub App.

**Comment by maxrake** [comment]
Closing as OBE due to #928 ...

**Pull request closed without merging**
